### PR TITLE
Update User Guide

### LINKED ISSUE #116: Update user guide

### PR DESCRIPTION
**Pull request by kimberlytmq**
Updated user guide to contain the details of recent feature updates. Screenshots have also been updated.

Closes #116

### CODE REVIEW

**Review by jtooya — DISMISSED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -16,62 +16,72 @@ their contacts into 'work' contacts and 'personal' contacts to better manage the
...
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+**3. Choose a Home Folder**
+- Move the `.jar` file to where you want to store your UniLink data. This will be your 'home folder'.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+**4. Launch the app!**
+- Open up Command Prompt (for Windows) or Terminal (for Mac) and navigate to the home folder with `cd path/to/your/home/folder`
```
Comment (achl1012) [inline docs/UserGuide.md]: Maybe the 'cd path/to/your/home/folder' could be changed to something like 'cd path-to-home-folder'? The multiple slashes may be a bit confusing for some people

**Review by achl1012 — CHANGES_REQUESTED** [review]
just a minor detail to consider, everything else looks good 😄

File: docs/UserGuide.md (hunk 1 above)
Comment (kimberlytmq) [reply docs/UserGuide.md]: I think the slashes need to be there cause if not some people may not know that you need to put slashes. The example is there too so I think it's clear? modified slightly

**Review by achl1012 — APPROVED** [review]
lgtm! 👍

**Pull request merged**